cpp/daal/include/services/internal/sycl/execution_context_sycl.h

```diff
@@ -297,26 +253,26 @@ class SyclExecutionContextImpl : public Base, public ExecutionContextIface
     }
 
     void copy(UniversalBuffer dest, size_t desOffset, UniversalBuffer src, size_t srcOffset, size_t count,
-              services::Status * status = nullptr) DAAL_C11_OVERRIDE
+              services::Status & status) DAAL_C11_OVERRIDE
     {
         DAAL_ASSERT(dest.type() == src.type());
         // TODO: Thread safe?
         try
         {
-            BufferCopier::copy(_deviceQueue, dest, desOffset, src, srcOffset, count);
+            BufferCopier::copy(_deviceQueue, dest, desOffset, src, srcOffset, count, status);
         }
         catch (cl::sycl::exception const & e)
         {
             convertSyclExceptionToStatus(e, status);
         }
     }
 
-    void fill(UniversalBuffer dest, double value, services::Status * status = nullptr) DAAL_C11_OVERRIDE
+    void fill(UniversalBuffer dest, double value, services::Status & status) DAAL_C11_OVERRIDE
     {
         // TODO: Thread safe?
         try
         {
-            BufferFiller::fill(_deviceQueue, dest, value);
+            BufferFiller::fill(_deviceQueue, dest, value, status);
```

Comment:
May by delete try-catch block in this case? The function always gets the status and reports the problems via this one instead of exceptions.

Reply:
This PR contains change only Status * -> Status &, the other thing will be in different PR